appsettings.py

```diff
@@ -3,7 +3,7 @@
 # Try to open the configuration file
 try:
     with open(
-        'test_config.json',
+        'appsettings.json',
         'r',
     ) as f:
         config = json.load(f)
```

Comment:
The error message still says test_config.json; can we swap that out with appsettings.json

Reply:
swapped with appsetting.json